engine/src/cmd/ai/navigation.cpp

```diff
@@ -684,7 +684,7 @@ void AutoLongHaul::Execute() {
         if (speed > .01) {
             cfacing = cfacing * (1. / speed);
         }
-        const float dotLimit = cos(3.1415926536 * configuration()->physics.auto_pilot_spec_lining_up_angle / 180.0);
+        const float dotLimit = cos(3.1415926536F * configuration()->physics.auto_pilot_spec_lining_up_angle_flt / 180.0F);
```

Comment:
Also here. In general, check all references to configuration in Execute functions. There's a good chance that's part of the slowdown.

Reply:
Should be fixed now